src/containers/Signin/index.js

```diff
@@ -1,6 +1,127 @@
 // This is a smart component which is aware of redux
-import React from 'react';
+import { bindActionCreators } from 'redux';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
-const Signin = () => <h1>Sign in page</h1>;
+import InputField from '../../components/InputField';
+import Errors from '../../components/Errors';
+import signinAction from './actions';
 
-export default Signin;
+class Signin extends Component {
+  static defaultProps = {
+    signin: {},
+  }
+
+  constructor(props) {
```

Comment:
You don't have to use this constructor since we're using arrow functions.

You can just declare the state by doing `state = {}`